app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -40,7 +40,12 @@
     <% if history.any? %>
       <div class="change-notes" data-module="toggle">
         <p>
-          <a href="#full-history" data-controls="full-history" data-expanded="false">
+          <a href="#full-history" data-controls="full-history"
+                                  data-expanded="false"
+                                  data-module="track-click"
+                                  data-track-category="content-history"
+                                  data-track-action="full-page-history-opened"
```

Comment:
This will track on open and close, might be worth saying: "full-page-history-toggled"